src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -145,11 +147,61 @@ const AccountTransfer = (() => {
         el_transfer_fee.setVisibility(0);
         el_success_form.setVisibility(1);
         getElementById('transfer_info').setVisibility(0);
+        getElementById('acc_transfer_header_icon').classList.add('success');
+        getElementById('acc_transfer_header_text').textContent = localize('Successfully transferred');
+    };
+
+    const onAmountInput = (e) => {
+        const input_val = parseFloat(e.target.value);
+        calculateAmount(input_val);
+    };
+
+    const calculateAmount = (value) => {
+        const el_amount    = getElementById('amount');
+        const el_amount_to = getElementById('amount_to');
+        const exchange_rate = getPropertyValue(exchange_rates, curr_account_to);
+        const transferred_amount = (value * exchange_rate).toFixed(8);
+        if (transferred_amount && !isNaN(transferred_amount)) {
+            el_amount_to.value = transferred_amount;
+        }
+        else {
+            el_amount_to.value = '';
+        }
+        const transfer_fee_value   = (parseFloat(el_amount.value) * 0.01);
+        const transfer_total_value = (parseFloat(el_amount.value) * 1.01);
+
+        elementTextContent(getElementById('transfer_fee_lbl'), `${client_currency} ${isCryptocurrency(client_currency) ? transfer_fee_value.toFixed(8) : transfer_fee_value.toFixed(2)}`);
```

Comment:
the decimal places shouldn't be hardcoded. you can get them from `Currency.getDecimalPlaces` function

Reply:
it will break for things like DAI which is crypto but only has 2 decimal places. better to get it from API.